templates/rule.epp

```diff
@@ -0,0 +1,13 @@
+<%- |
+  Variant[Array[String[1]], String[1]] $content
+|-%>
+<%
+  if $content =~ Array {
+-%>
+<%= join(strip($content).filter|$x| { !empty($x) }, "\n") %>
```

Comment:
I didn't see a test for this for auditd::rule (Array input) nor did I see a use of auditd::rule with Array content throughout the module.  Please add a test.